lte/gateway/c/core/oai/tasks/amf/amf_app_state_converter.cpp

```diff
@@ -621,6 +689,30 @@ void AmfNasStateConverter::proto_to_qos_flow_setup_request_item(
       qos_flow_item_proto.qos_flow_param(),
       &state_qos_flow_request_item->qos_flow_level_qos_param);
 }
+
+void AmfNasStateConverter::smf_context_map_to_proto(
+    const std::unordered_map<uint8_t, std::shared_ptr<smf_context_t>>
```

Comment:
```suggestion
    const std::unordered_map<uint8_t, std::shared_ptr<smf_context_t>>&
```
pass by const reference

Reply:
Addressed Review Comment